webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -50,71 +50,24 @@ const TasksTablePage = ({
     }));
```

Comment:
It's a function thats only called when needed in the modal, instead of always sending the array of selected objects. are you sure this should be moved?

Reply:
I see it can be useful for the confirm actions, nvm